converter/fluka/cards/figure_card.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import dataclass, field
+from converter.common import format_float
+
+from converter.fluka.helper_parsers.figure_parser import FlukaBox, FlukaCylinder, FlukaFigure, FlukaSphere
+
+
+@dataclass
+class FiguresCard:
+    """Class representing description of figures in Fluka input"""
+
+    data: list[FlukaFigure] = field(default_factory=lambda: [])
+
+    def __str__(self) -> str:
+        """Return the card as a string."""
+        result = ""
+        for index, figure in enumerate(self.data):
+            if index == 0:
+                line = ""
+            else:
+                line = "\n"
+            line += f"{figure.figure_type} {figure.name}"
+            if type(figure) is FlukaBox:
+                line += (f" {format_float(figure.x_min, n=16):+#}"
+                         f" {format_float(figure.x_max, n=16):+#}"
+                         f" {format_float(figure.y_min, n=16):+#}"
+                         f" {format_float(figure.y_max, n=16):+#}"
+                         f" {format_float(figure.z_min, n=16):+#}"
+                         f" {format_float(figure.z_max, n=16):+#}")
+            elif type(figure) is FlukaCylinder:
+                line += (f" {format_float(figure.coordinates[0], n=16):+#}"
+                         f" {format_float(figure.coordinates[1], n=16):+#}"
+                         f" {format_float(figure.coordinates[2], n=16):+#}"
+                         f" {format_float(figure.height_vector[0], n=16):+#}"
+                         f" {format_float(figure.height_vector[1], n=16):+#}\n"
+                         f"{format_float(figure.height_vector[2], n=16):+#}"
+                         f" {format_float(figure.radius, n=16):+#}")
```

Comment:
Can you add here a line of comment as well, so instead of:
```
RCC fig2 +0.0 +0.0 -0.5 +0.0 +0.0
+1.0 +4.0
```

we should have

```
* cylinder fig2, bottom center (0,0,-0.5), spanning vector (0,0,1), 
* radius 4, height XX cm
* rotation angles: X*, Y*, Z*
RCC fig2 +0.0 +0.0 -0.5 +0.0 +0.0
+1.0 +4.0
```

information may be redundant, but this way we increase readability. 
Same for other figures.

Reply:
ok, I made these changes